src/api/ledgerStorage/github.js

```diff
@@ -67,7 +89,75 @@ export class GithubStorage implements LedgerStorage {
     return Ledger.parse(rawLedger);
   }
 
-  async write() {
-    throw new Error("method not implemented");
+  async write(ledger: Ledger) {
+    const ledgerData = ledger.serialize();
+
+    // Get latest commit hash of target branch
+    const refRes = await fetch(
```

Comment:
I strongly prefer verbose naming conventions. `refRes` and other similar names throughout don't really communicate to me what the variable is. I can guess, but the ambiguity and added cognitive load make variable names like this less maintainable.